### PR TITLE
Fix NRF reader test

### PR DESCRIPTION
**Pull request by sebwolf-de**
Was broken by https://github.com/SeisSol/SeisSol/commit/29ce470d0e5bb216e645203c6028deb5d44a83db

### CODE REVIEW

File: src/tests/Reader/NRFReader.t.h [hunk 1]
```diff
@@ -31,7 +31,6 @@ TEST_CASE("NRF Reader") {
   REQUIRE(nrf.subfaults[0].tinit == AbsApprox(0.0));
   REQUIRE(nrf.subfaults[0].timestep == AbsApprox(0.0002));
   REQUIRE(nrf.subfaults[0].mu == AbsApprox(0.0));
-  REQUIRE(nrf.source == 1);
```
Comment (davschneller) [inline src/tests/Reader/NRFReader.t.h:34]: `REQUIRE(nrf.size() == 1);`

Maybe also test that `nrf.subfaults.size()==nrf.centres.size()==nrf.sroffsets.size()` (the syntax used in this comment here is of course not correct)

**Review by davschneller — COMMENTED** [review]
Thanks! However, I'd not remove the assert, but rather update it as well.

That brings up the question in more general ... why is Netcdf disabled for the tests right now?

**Comment by sebwolf-de** [comment]
That's a good question. The CI docker container explicitely contains netcdf. I just enabled it for the tests.

**Comment by davschneller** [comment]
I fear that's the wrong pipeline still (it's the `Testing_SeisSol_CPU` one); the reader tests are still not executed.

For the unit tests, cf. https://github.com/SeisSol/SeisSol/blob/master/azure-pipelines-seissol-template.yml#L64

**Review by davschneller — APPROVED** [review]
Ok, approved.

Though I guess long-term (and if feasible), it would be best to integrate all unit tests in either the CPU runner or Azure.

**Pull request merged**